template_app/package.json

```diff
@@ -6,11 +6,16 @@
     "dev": "vite dev",
     "build": "vite build",
     "preview": "vite preview",
-    "test": "npm run test:integration && npm run test:unit",
+    "test": "pnpm run test:prep && pnpm run test:all",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "lint": "prettier --check . && eslint .",
```

Comment:
Perhaps worth adding a test-rust task in here to run "cargo test" in the rust lib

Reply:
That looks pretty easy. Would you rather than's in `template_app/package.json` or `package.json` (or both...?).

Reply:
Stuck it in `template_app/package.json`, so propagates to `package.json`.